docs/components/board/customlinux.md

```diff
@@ -213,15 +230,104 @@ On your `customlinux` board, create a file in your <file>/home/root</file> direc
+To see which chips exist and how many lines each chip has, run this command on your board:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo gpiodetect
+```
+
+Here is example output from this command on an Odroid C4:
+
+```sh {id="terminal-prompt" class="command-line" data-output="1-10"}
+gpiochip0 [aobus-banks] (16 lines)
+gpiochip1 [periphs-banks] (86 lines)
+```
+
+This example output indicates that there are two GPIO chips on this board.
+One has 16 lines, numbered 0-15.
+The other has 86 lines, numbered 0-85.
+
+To see info about every line on every GPIO chip, run this command:
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo gpioinfo
+```
+
+Here is example output from the `sudo gpioinfo` command on an Odroid C4:
+
+```sh {id="terminal-prompt" class="command-line" data-output="1-10"}
+...
+  line  62:      unnamed       unused   input  active-high
+  line  63:      unnamed       unused   input  active-high
+  line  64:     "PIN_27"       unused   input  active-high
+  line  65:     "PIN_28"       unused   input  active-high
+  line  66:     "PIN_16"       unused   input  active-high
+  line  67:     "PIN_18"       unused   input  active-high
+  line  68:     "PIN_22"       unused   input  active-high
+...
+```
+
+In this example, the human-readable names such as `"PIN_28"` indicate which physical board pin each line is attached to.
+However, these names are not standardized.
+Some boards have pin names like `"PH.03"`.
+In either case, you need to look at the data sheet for your board and determine how the pin names map to the hardware.
+
+### Tips for finding PWM information
+
+Run the following command and look for unique strings within each [symlink](https://en.wikipedia.org/wiki/Symbolic_link).
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+ls -l /sys/class/pwm
+```
+
+Here is example output from this command on a Jetson Orin AGX:
+
+```sh {id="terminal-prompt" class="command-line" data-output="1-10"}
+total 0
+lrwxrwxrwx 1 root root 0 Sep  8  2022 pwmchip0 -> ../../devices/platform/3280000.pwm/pwm/pwmchip0
+lrwxrwxrwx 1 root root 0 Sep  8  2022 pwmchip1 -> ../../devices/platform/32a0000.pwm/pwm/pwmchip1
+lrwxrwxrwx 1 root root 0 Sep  8  2022 pwmchip2 -> ../../devices/platform/32c0000.pwm/pwm/pwmchip2
+lrwxrwxrwx 1 root root 0 Sep  8  2022 pwmchip3 -> ../../devices/platform/32f0000.pwm/pwm/pwmchip3
+lrwxrwxrwx 1 root root 0 Sep  8  2022 pwmchip4 -> ../../devices/platform/39c0000.tachometer/pwm/pwmchip4
+```
+
+Based on this example output, the values to use for `pwm_chip_sysfs_dir` are `328000.pwm`, `32a0000.pwm`, and so on.
+
+Each of these directories contains a file named <file>npwm</file> containing a number.
+The number in each file is the number of lines on the chip.
+The `pwm_id` value will be between `0` and one less than the number of lines.
+For example, if the <file>npwm</file> contains `"4"`, then the valid `pwm_id` values are `0`, `1`, `2`, and `3`.
```

Comment:
:+1: Thanks for including the example! That makes it clear that it's between 0 and one less than the npwm value, inclusive.

Reply:
Thank you for providing this example on the ticket!!